frontend/src/icons/star.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const StarIcon = ({ width, color }) => {
```

Comment:
Please create a generic component that takes the `width`, `color`, and `path` as props to build a generic colored svg for different icons